yb-voyager/src/srcdb/pg_dump_extract_schema.go

```diff
@@ -18,7 +18,10 @@ func pgdumpExtractSchema(schemaList string, connectionUri string, exportDir stri
 
 	cmd := fmt.Sprintf(`pg_dump "%s" --schema-only --schema "%s" --no-owner -f %s --no-privileges --no-tablespaces`,
 		connectionUri, schemaList, filepath.Join(exportDir, "temp", "schema.sql"))
-	log.Infof("Running command: %s", cmd)
+	redactedUri := utils.GetRedactedURLs([]string{connectionUri})[0]
+	redactedCmd := fmt.Sprintf(`pg_dump "%s" --schema-only --schema "%s" --no-owner -f %s --no-privileges --no-tablespaces`,
```

Comment:
Same as above.

Reply:
Do we log ora2pg command line in a similar manner?

Reply:
For ora2pg, all the config options are edited into the ora2pg.conf template file, the shell command itself contains no sensitive information